source/isaaclab/isaaclab/sensors/utils.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2022-2025, The Isaac Lab Project Developers (https://github.com/isaac-sim/IsaacLab/blob/main/CONTRIBUTORS.md).
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+from __future__ import annotations
+
+import torch
+
+import omni.physics.tensors.impl.api as physx
+from isaacsim.core.prims import XFormPrim
+
+from isaaclab.utils.math import convert_quat
+
+
+def compute_world_poses(
```

Comment:
Instead of opening a new `utils.py`, can we just move these to `sim/utils.py` and rename it to `obtain_world_pose_from_view` or something?